testing/utils/openshift_ci/pr_args.sh

```diff
@@ -89,11 +89,12 @@ else
     last_comment_page_json=$(curl -sSf "$PR_LAST_COMMENT_PAGE_URL")
 fi
 
+REQUIRED_AUTHOR=$(jq -r .user.login <<< "$pr_json")
 REQUIRED_AUTHOR_ASSOCIATION=CONTRIBUTOR
 
 echo "PR comments URL: $PR_COMMENTS_URL" >&2
 last_user_test_comment=$(echo "$last_comment_page_json" \
-                             | jq '.[] | select(.author_association == "'$REQUIRED_AUTHOR_ASSOCIATION'") | .body' \
+                             | jq '.[] | select(.author_association == "'$REQUIRED_AUTHOR_ASSOCIATION'"), select(.user.login == "'$REQUIRED_AUTHOR'") | .body' \
```

Comment:
thanks @mcharanrm :)

can you modify the next line to have `(grep "^$test_anchor" || true)` so that the test anchor is matched only at the beginning of the line?

and in the error message a few lines after, something like this:

```
echo "ERROR: last comment of from a '$REQUIRED_AUTHOR_ASSOCIATION' or author=$REQUIRED_AUTHOR could not be found (searching for '$test_anchor') ..." >&2
```

Reply:
e9599d2f4a23b27e146f2eb56ef59423c08d13db , includes AUTHOR information in the error message.